### DISCUSSION TITLE
Simulation of calculated and imported Route

### DISCUSSION THREAD

**Discussion by Mike-0815** [organicmaps/organicmaps#7365] [Ideas]
Hello, I would be very happy if OM would offer the possibility to SIMULATE driving or walking along 1) calculated and 2) imported routes. This would be very helpful, as this functionality would allow the behavior of OM to be tested from home. I really hope to see this one day in OM, because I really miss it.

**Comment by biodranik** [comment]
Is it #5827  ?

**Comment by Mike-0815** [comment]
Yes. Also there's another thread/conversation of the same wish. But both are closed, mine is the only open at the moment I think.
